src/main/kotlin/com/github/jyoo980/reachhover/ui/ReachabilityButton.kt

```diff
@@ -55,6 +56,12 @@ sealed class ReachabilityButton(
             "<span style=\"font-family:JetBrains Mono;\">$it</font></span>"
         }
     }
+
+    private fun questionText(): String {
+        val inspectedName = identifierName() ?: "this argument"
+        val key = if (dataflowFromHere) "modifiedQuestion" else "createdQuestion"
+        return "<html>${MyBundle.message(key, inspectedName)}</html>"
+    }
```

Comment:
The addition of this method probably makes it easier to reuse logic in the `setText` method we have in the subtypes. I'll take a closer look at this later.